src/main/java/de/teamlapen/vampirism/entity/hunter/HunterTrainerEntity.java

```diff
@@ -75,6 +78,22 @@ public void setHome(AxisAlignedBB box) {
         this.setMoveTowardsRestriction(MOVE_TO_RESTRICT_PRIO, true);
     }
 
+    @Override
+    public void readAdditional(CompoundNBT nbt) {
+        super.readAdditional(nbt);
+        if(nbt.contains("createHome") && (this.shouldCreateHome = nbt.getBoolean("createHome"))){
```

Comment:
Does this need to happen on load, is it not possible to set the home on entity creation?

Should "createHome" stay true after setting the home pos?

Reply:
1. not possible, because the entity is loaded from NBT (as save file), which results in not create a new Entity
2.1. the entity is loaded and saved two times before joining ?! -> if not saved the home is not created the second time.
2.2. if the home is set, it is saved by HunterEntity and on the next load the home will not be created new